src/constants/enums/cmsDivisionsAndOffices.js

```diff
@@ -0,0 +1,117 @@
+const cmsDivisionsAndOffices = [
```

Comment:
For now, keeping this cached on the client. If we want to move this to the backend, we'd need to make an API call to retrieve this data. Depending on the form/page, we might need to make a handful of calls to get all relevant data.